Sources/SwiftFileStore/FileObjectStore.swift

```diff
@@ -70,6 +70,24 @@ public final class FileObjectStore: ObjectStore {
     }
     return try await removeAllTask.value
   }
+    
+    public func readAll(namespace: String) async throws -> [String] {
```

Comment:
should this be renamed to `readAllKeys` instead?

Reply:
also just curious why do you need this functionality? I designed `ObjectStore` more as a key-value store so caller should already know what are the keys for each object.